azure-webapp-maven-plugin/README.md

```diff
@@ -1,186 +1,179 @@
-
-    ```
-    $ az login
-    $ az account set --subscription <subscription Id>
+#### `azure-webapp:config`
+-  A plugin goal providing a great experience for managing the configuration of [azure-webapp:deploy](#azure-webappdeploy), it will rewrite your pom configuration based on your input. Details about the configuration could be found in the following.
+-  The plugin goal will divide the configuration into 3 parts.
+   1. Application
+        
+        You can configure appName, resource group, region, pricing tier at this part.
+   2. Runtime
+        
+        You can configure operating system and runtime stack at this part. It should be noted that you can not change the operating system after you deploy it.
+   3. DeploymentSlot
+        
+        You can configure deployment slot at this part.
+
+#### `azure-webapp:deploy`
+- Deploy artifacts or docker container image to an Azure Web App based on your configuration.<br>If the specified Web App does not exist, it will be created.
+- Configuration:
+    Common configurations of all Maven Plugins for Azure can be found [here](../docs/common-configuration.md).    
+
+    > This maven plugin supports two kinds of configurations V2 and V1 (deprecated). Using v2 schema is recommended. Please click [here](v1-schema.md) if you want to find more details about v1 schema (deprecated).
+
+    Property | Required | Description | Version
+    ---|---|---|---
+    `<schemaVersion>` | false | Specify the configuration `<schemaVersion>V2</schemaVersion>` to use the V2 configuration. | 1.5.2
+    [`<region>`](#region) | true | Specifies the region where your Web App will be hosted; the default value is **westus**. All valid regions at [Supported Regions](#region) section. | 0.1.0+
+    `<resourceGroup>` | true | Azure Resource Group for your Web App. | 0.1.0+
+    `<appName>` | true | The name of your Web App. | 0.1.0+
+    [`<pricingTier>`](#pricingtier) | false | The pricing tier for your Web App. The default value is **P1V2**.| 0.1.0+
+    `<deploymentSlot>` | false | The deployment slot to deploy your application. You could find detail samples for deployment slot at [here](../docs/web-app-samples.md)| 1.3.0+
+    `<appServicePlanResourceGroup>` | false | The resource group of the existing App Service Plan. If not specified, the value defined in `<resourceGroup>` will be used by default. | 1.0.0+
+    `<appServicePlanName>` | false | The name of the existing App Service Plan. | 1.0.0+
+    `<appSettings>` | false | Specifies the application settings for your Web App. You could find detail samples for appsettings at [here](../docs/web-app-samples.md). | 0.1.0+
+    `<stopAppDuringDeployment>` | false | To stop the target Web App or not during deployment. This will prevent deployment failure caused by IIS locking files. | 0.1.4+
+    [`<runtime setting>`](#runtimesetting) | true | The runtime environment configuration, you could see the detail [here](#runtimesetting) | 0.1.0+
```

Comment:
Make sure all the sentences here have the same style.

For example: all of them having `.` at last. Or not having `.` at all.